hasktorch/hasktorch.cabal

```diff
@@ -96,10 +96,11 @@ library
  other-modules:       Paths_hasktorch
  hs-source-dirs:      src
  default-language:    Haskell2010
- ghc-options:         -fplugin GHC.TypeLits.Normalise -fplugin GHC.TypeLits.KnownNat.Solver -fplugin GHC.TypeLits.Extra.Solver -fconstraint-solver-iterations=0
+ ghc-options:         -fplugin GHC.TypeLits.Normalise -fplugin GHC.TypeLits.KnownNat.Solver -fplugin GHC.TypeLits.Extra.Solver -fconstraint-solver-iterations=0 -fplugin GHC.NotExport.Plugin
```

Comment:
this seems like a typo -- shouldn't this only be on libtorch-ffi?

Reply:
It is not a typo.
Torch.Vision uses inline-c, so I add it.

Reply:
Another module will use inline-c.
It is a preventive measure.
https://github.com/hasktorch/hasktorch/pull/600